components/tools/OmeroWeb/omeroweb/webclient/show.py

```diff
@@ -685,7 +685,8 @@ def paths_to_object(conn, experimenter_id=None, project_id=None,
             select coalesce(sowner.id, plowner.id, aowner.id, wsowner.id),
                    slink.parent.id,
                    plate.id,
-                   acquisition.id
+                   acquisition.id,
+                   well.id
             from WellSample wellsample
             left outer join wellsample.details.owner wsowner
```

Comment:
We need the owner of the top element in the path. E.g. Screen OR Plate. Probably never need Acquisition or WellSample. Even if lower elements in the path are owned by a different owner, that doesn't matter. We just need to know the context for browsing the jsTree, starting at Experimenter -> Screen or Experimenter -> Plate.